packages/example-app-base/src/components/SideRail/SideRail.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
+import { css, FocusZone, FocusZoneDirection, Link, IProcessedStyleSet, classNamesFunction, styled } from 'office-ui-fabric-react';
+import { isPageActive, jumpToAnchor, getPathMinusLastHash } from '../../utilities/index2';
+import { ISideRailProps, ISideRailLink, ISideRailStyles, ISideRailStyleProps } from './SideRail.types';
+import { getStyles, sideRailClassNames } from './SideRail.styles';
+
+export interface ISideRailState {
```

Comment:
Should not export state

Reply:
We have to export state in all of our components because we build with `declaration: true`.